setup.py

```diff
@@ -1,20 +1,20 @@
+import six
```

Comment:
We can't import six in this file, since we're not guaranteed the system will have it (that said, its extremely unlikely someone wouldn't have six installed at this point but alas). This said you have a few options;

- Use the builtin strategy of detecting the Python version. I'm ok with this.
- Split out earl-etf into a separate extras (performance-py3 ?). Also ok with this.